infra/templates/azure-simple/README.md

```diff
@@ -30,13 +30,13 @@ Application developer team that want to provision a secure app service plan in a
 
 - An Azure subscription
 - A [service principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)
-- An azure storage account for tracking terraform remote backend state. You can use our backend state setup [template]((/infra/templates/backend-state-setup/README.md)) to provision the storage resources.
+- An Azure storage account for tracking terraform remote backend state. You can use our backend state setup [template](/infra/templates/backend-state-setup/README.md) to provision the storage resources.
 - Install [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html)
-- Local environment [setup](https://github.com/microsoft/cobalt/blob/erisch/features/bashwizard/test-harness/README.md#local-environment-setup)
+- Local environment [setup](https://github.com/microsoft/cobalt/tree/master/test-harness#local-environment-setup)
```

Comment:
FYI, a local reference will be more robust to repo changes and will also refer you to the right file if it changes as part of your branch